cli/crates/cli/tests/graphql-directive/server.rs

```diff
@@ -64,7 +64,7 @@ impl Query {
             }));
         } else if id == "3" {
             return Some(PullRequestOrIssue::Issue(Issue {
-                title: "Everythings fine".into(),
```

Comment:
This kind of thing is why I have reservations about installing this tool on CI.

This is "wrong" but in a way that doesn't matter in the slightest.  I'd get extremely annoyed if I had to wait another 30 minutes on CI because it caught this.

Reply:
I commented in Slack, it would be better as a periodic check since it has false positives in the engine. But we should run it occasionally

Reply:
Note that most of these were automatic fixes